File: DSA/VenikaPanwar_590013755/Day_29_Question_3.java
import java.util.*;

public class Day_29_Question_3 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter the number (n): ");
        int n = sc.nextInt();

        System.out.print("Enter the bit position to set (i): ");
        int i = sc.nextInt();

        int mask = 1 << i;
        int result = n | mask;

        System.out.println("New number after setting bit " + i + ": " + result);
    }
}

File: DSA/Ayush_Mishra_590014844/Day_29_Question_3.c
#include <stdio.h>

int main() {
    int n, i;
    printf("Enter the number (n): ");
    scanf("%d", &n);
    printf("Enter the bit position to set (i): ");
    scanf("%d", &i);
    int result = n | (1 << i);
    printf("Number after setting %d-th bit: %d\n", i, result);
    return 0;
}
